docs/ipr/scripting/RNA_Expression_Metrics.ipynb

```diff
@@ -3926,121 +3969,59 @@
-              "ENSG00000256162.2   -9.9658 -1.0559  ...         2.0    reduced expression\n",
-              "ENSG00000205209.7   -6.5064 -3.4580  ...         2.0    reduced expression\n",
-              "ENSG00000204542.2    3.6382 -9.9658  ...        98.0  increased expression\n",
-              "ENSG00000166961.14   2.4359 -9.9658  ...       100.0  increased expression\n",
-              "ENSG00000102096.9    5.7455  2.5112  ...        98.0  increased expression\n",
-              "ENSG00000124191.17   5.1023  0.2522  ...        98.0  increased expression\n",
-              "ENSG00000107485.15   3.9175 -1.9379  ...       100.0  increased expression\n",
-              "ENSG00000065618.16   5.4741 -4.6082  ...        98.0  increased expression\n",
-              "ENSG00000158816.15  -0.1828 -9.9658  ...       100.0  increased expression\n",
-              "ENSG00000135374.9    3.6077 -9.9658  ...       100.0  increased expression\n",
-              "ENSG00000145113.21   5.3618 -2.5479  ...       100.0  increased expression\n",
-              "ENSG00000162367.11   2.7847 -1.5522  ...       100.0  increased expression\n",
-              "ENSG00000105371.8    1.3735 -3.8160  ...        98.0  increased expression\n",
-              "ENSG00000070526.14   4.1252 -2.7274  ...        98.0  increased expression\n",
-              "\n",
-              "[57 rows x 8 columns]"
             ]
           },
           "metadata": {
             "tags": []
           },
           "execution_count": 56
         }
-      ]
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 1000
+        },
+        "id": "sIhQtsBe13xb",
+        "outputId": "8453cb4f-3d0a-494c-824c-fc6ab5062078"
+      }
     },
     {
       "cell_type": "markdown",
-      "metadata": {
-        "id": "WoGtfcS63p8N"
-      },
       "source": [
         "Finally we can use this to create our output. The percentiles and kIQR values we have calculated above were for our disease comparator but a similar process would be used for normal and biopsy comparators"
-      ]
+      ],
+      "metadata": {
+        "id": "WoGtfcS63p8N"
+      }
     },
     {
       "cell_type": "code",
-      "metadata": {
-        "id": "_QSgxMml4Kw8"
-      },
+      "execution_count": 57,
       "source": [
         "metrics_df = metrics_df.rename(columns={'kiqr': 'diseasekIQR', 'percentile': 'diseasePercentile'}).reset_index()\n"
       ],
-      "execution_count": 57,
-      "outputs": []
+      "outputs": [],
+      "metadata": {
+        "id": "_QSgxMml4Kw8"
+      }
     },
     {
       "cell_type": "markdown",
-      "metadata": {
-        "id": "0H55j7Vw5BKe"
-      },
       "source": [
         "We often leave non-variant expression records in to supplement the copy variant and structural variant information, for these records the kbCategory field is a null value since they are not an expression outlier"
```

Comment:
There should probably be a period here instead of a comma to break up the sentence